### PR TITLE
Recruitment

### PR DESCRIPTION
**Pull request by Annnnnieee**

### CODE REVIEW

**Review by Schemetrical — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -17,6 +17,7 @@
   <link href='https://fonts.googleapis.com/css?family=Lato:300,400' rel='stylesheet' type='text/css'>
   <link href='https://fonts.googleapis.com/css?family=Open+Sans:300,400,700' rel='stylesheet' type='text/css'>
   <link href="https://fonts.googleapis.com/css?family=Lato&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Lato&display=swap" rel="stylesheet">
```
Comment (Schemetrical) [inline index.html]: Duplicate?

**Review by Schemetrical — CHANGES_REQUESTED** [review]

File: index.html [hunk 2]
```diff
@@ -54,7 +55,8 @@ <h1>
   <!-- BANNER -->
   <div id="banner">
     <div class="banner-brand">
-      <video preload loop muted autoplay class="banner-video">
+      <video playsinline preload loop muted autoplay class="banner-video">
+        <source src="/media/background-video.mp4" type="video/mp4">
         <source src="/media/background-video.webm" type="video/webm">
```
Comment (Schemetrical) [inline index.html:59]: Either webm or mp4

**Review by Schemetrical — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Annnnnieee) [reply index.html]: oops!

File: index.html (hunk 2 above)
Comment (Annnnnieee) [reply index.html:59]: having both allows the browser to choose which to use. So just keeping mp4 makes it stop working on firefox, but just keeping webm makes it stop working in safari. this works on mobile as well, except on iphones >_>, not sure why.

**Review by Schemetrical — APPROVED** [review]

**Pull request merged**